src/FakeRTCDataChannel.ts

```diff
@@ -298,7 +298,7 @@ export class FakeRTCDataChannel extends EventTarget implements RTCDataChannel
 				case 'message':
 				{
 					// @ts-ignore
-					this.dispatchEvent(new Event('message', { data }));
+					this.dispatchEvent(new MessageEvent('message', { data }));
```

Comment:
Shouldn't we import MessageEvent from the same lib from which we import Event?

Reply:
`event-target-shim` does not provide MessageEvent: https://github.com/mysticatea/event-target-shim/blob/master/src/index.ts
So I couldn't import it.
And MessageEvent is included in Node.js, so we can compile and execute this code without import.
https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent